scripts/fbi/crime/preprocess.py

```diff
@@ -111,15 +147,15 @@ def _int_from_field(f):
         return 0
 
 
-# When there is two entries for the same city, skip the one with wrong population data.
-def _should_skip_special_line(year, field):
-    if year == '2019' and field[_STATE_INDEX] == 'OREGON' and field[
-            _CITY_INDEX] == 'Ashland' and field[_POPULATION_INDEX] == '2670.0':
-        return True
-    if year == '2019' and field[_STATE_INDEX] == 'OREGON' and field[
-            _CITY_INDEX] == 'Lebanon' and field[_POPULATION_INDEX] == '25959.0':
-        return True
-    return False
+# Trim 2016 state name.
```

Comment:
What was the issue before that required custom population handling? Can we update the README which still talks about duplicate city population (https://github.com/datacommonsorg/data/blob/master/scripts/fbi/crime/README.md)